### PR TITLE
Update RBMA Radio URL

### PR DESCRIPTION
**Pull request by StevenMaude**
It's been rebranded as "Red Bull Radio".

## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [X] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [X] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [X] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [X] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

Fix downloads from Red Bull Radio (was Red Bull Music Academy).

They weren't being recognised by the extractor regex as the URL has changed.

Current master (90e3f18fc1cff2c12361cbbd85303774bf7d59f7) falls back to generic information
extractor and fails with "Unsupported URL".

With this change, downloading works again.

### CODE REVIEW

**Comment by StevenMaude** [comment]
First time looking at source here, not sure how the info_dict for test files are generated.

The [test download](https://www.redbullradio.com/shows/main-stage/episodes/ford-lopatin-live-at-primavera-sound-2011) works though I get a different md5sum for the downloaded file (1146f77f34ee2995101d86d26d61eb1b not 6bc6f9bcb18994b4c983bc3bf4384d95).

File: youtube_dl/extractor/rbmaradio.py [hunk 1]
```diff
@@ -13,15 +13,15 @@
 
 
 class RBMARadioIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?rbmaradio\.com/shows/(?P<show_id>[^/]+)/episodes/(?P<id>[^/?#&]+)'
+    _VALID_URL = r'https?://(?:www\.)?redbullradio\.com/shows/(?P<show_id>[^/]+)/episodes/(?P<id>[^/?#&]+)'
```
Comment (dstftw) [inline youtube_dl/extractor/rbmaradio.py]: Do not remove the old domain.

Comment (StevenMaude) [inline youtube_dl/extractor/rbmaradio.py]: Sorry, unfamiliar with this code.

Fixed, and force pushed a cleaned version of the branch.

File: youtube_dl/extractor/rbmaradio.py [hunk 2]
```diff
@@ -13,15 +13,15 @@
 
 
 class RBMARadioIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?rbmaradio\.com/shows/(?P<show_id>[^/]+)/episodes/(?P<id>[^/?#&]+)'
+    _VALID_URL = r'https?://(?:www\.)?(rbmaradio|redbullradio)\.com/shows/(?P<show_id>[^/]+)/episodes/(?P<id>[^/?#&]+)'
```
Comment (dstftw) [inline youtube_dl/extractor/rbmaradio.py]: Do not capture groups you don't use.

Comment (StevenMaude) [inline youtube_dl/extractor/rbmaradio.py]: Fixed.

**Pull request merged**